docs/types/operators.rst

```diff
@@ -26,6 +26,18 @@ except for comparison operators where the result is always ``bool``.
 The operators ``**`` (exponentiation), ``<<``  and ``>>`` use the type of the
 left operand for the operation and the result.
 
+.. note::
```

Comment:
I'm sorry - I just now opened this page in a wider context, and I think this should just be a section
```
Ternary Operator
--------------
```
instead of a note.